README.md

```diff
@@ -8,7 +8,7 @@ After [installation](#install-ucx), ensure to [trigger](#ensure-assessment-run-c
 so that you'll be able to [scope the migration](docs/assessment.md) and execute the [group migration workflow](#group-migration-workflow).
 
 The [README notebook](#readme-notebook), which can be found in the installation folder contains further instructions and explanations of the different ucx workflows & dashboards.
-Once the migration is scoped, you can start executing the [table migration workflow](#table-migration-workflow).
+Once the migration is scoped, you can start executing the [table migration workflow](#Table-Migration).
```

Comment:
Let's change the "table migration workflow" to "table migration process" as it is more than just one workflow like the assessment:

```suggestion
Once the migration is scoped, you can start with the [table migration process](#Table-Migration).
```

Can you update the others as well?

Reply:
Done